custom/icds/rules/custom_criteria.py

```diff
@@ -61,6 +61,28 @@ def person_case_is_under_N_years_old(case, now, n_years):
     return todays_date(now) < (dob + relativedelta(years=n_years))
 
 
+def icds_ccs_record_case_has_future_edd_and_null_add(case, now):
+    """
+    NOTE: This criteria references today's date.
+    Use this custom criteria with caution for SMS alerts.
+    See `person_case_is_under_n_years_old` for explanation.
+    """
+    if case.type != 'ccs_record':
+        return False
+
+    try:
+        edd = get_date(case.get_case_property('edd'))
+    except Exception:
+        return False
+
+    try:
+        add = case.get_case_property('add')
+    except Exception:
+        add = None
```

Comment:
I'd rather not include the check for "add is null" in this because that's something we can handle with normal, out of the box criteria in the rule. the only custom part that we need is checking that today < edd

Reply:
326807e21822c2046637cc95095ef8c9c2d2e3d7